deploy/helm/edot-collector/kube-stack/managed_otlp/values.yaml

```diff
@@ -16,7 +16,7 @@ crds:
   create: true # Install the OpenTelemetry Operator CRDs.
 defaultCRConfig:
   image:
-    repository: "docker.elastic.co/beats/elastic-agent"
+    repository: "docker.elastic.co/beats/elastic-otel-collector"
```

Comment:
@pkoutsovasilis do we have test for helm charts?

Reply:
So there are already some that utilise the EDOT variant [here](https://github.com/elastic/elastic-agent/blob/main/testing/integration/otel_helm_test.go#L31) and [here](https://github.com/elastic/elastic-agent/blob/main/testing/integration/kubernetes_agent_standalone_test.go#L206)

and there is also [this issue](https://github.com/elastic/elastic-agent/issues/7381) to bring in more 🙂